dipy/viz/tests/test_actors.py

```diff
@@ -103,12 +103,18 @@ def test_slicer():
                                       value_range=(0., 1.))
     renderer.clear()
     slicer_lut = actor.slicer(data, lookup_colormap=lut)
-
     slicer_lut.display(10, None, None)
     slicer_lut.display(None, 10, None)
     slicer_lut.display(None, None, 10)
-
+    slicer_lut.opacity(0.5)
+    slicer_lut.tolerance(0.03)
```

Comment:
Since we're explicitly testing copy starting next line, it's preferable to add a newline after this.

Reply:
Done